docs/documentation/models.md

```diff
@@ -22,8 +22,8 @@ npm i @axflow/models
 - ✅ Anthropic and Anthropic-compatible Completion models
 - ✅ HuggingFace text generation inference API and Inference Endpoints
 - ✅ Ollama.ai models running generation or embedding generation
+- ✅ Azure OpenAI
```

Comment:
Can we add a link below to the page (where the other links are)

Reply:
Looks like the shared link needs to be fixed too

Reply:
What's the shared link?

Reply:
The link to the `shared` module, the only one that is formatted differently than all the others

Reply:
fixed (separate commit, on main)